lab4/lvl3/lab4.py

```diff
@@ -0,0 +1,38 @@
+def read_from_file(file_name: str):
+    with open(file_name, "r") as file:
+        n = int(file.readline().strip())
+        print(n)
```

Comment:
remove this print unless it is intended and helps better understand the code (this is the same for other `print`'s too)